src/psyclone/dynamo0p3.py

```diff
@@ -847,6 +848,73 @@ def __str__(self):
         return res
 
 
+class RefElementMetaData(object):
```

Comment:
When running `pylint` with Python 3, it returns `R0205: Class 'RefElementMetaData' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)`. I do not think we need to do anything about it here, but it would be good to keep it in mind for when we no longer support Python 2.

Reply:
Yes, it's a pain but one day we can get rid of it :-)